source/nodejs/adaptivecards-designer/src/card-designer.ts

```diff
@@ -1409,6 +1503,17 @@ export class CardDesigner extends Designer.DesignContext {
         this.buildDataExplorer();
     }
 
+    get hostDataStructure(): FieldDefinition {
+        return this._hostDataStructure;
+    }
+
+    set hostDataStructure(value: FieldDefinition) {
+        this._hostDataStructure = value;
+
+        // Question: Do we need a host data explorer?
```

Comment:
From what I can tell, it looks like we were planning on deprecating the data toolbox. Is that correct?